### PR TITLE
removed creds from travis.config.yml and added a condition to test for n...

### PR DESCRIPTION
**Pull request by LaelWatkins**
the prod passwords have been reset and removed from config.travis.yml

### CODE REVIEW

File: app/models/insight_analyses.rb [hunk 1]
```diff
@@ -12,9 +12,11 @@ def increase_recommendation_contacts(designation_number)
     recommends = increase_recommendation_analysis(designation_number)
     contacts_list = []
     unless recommends.empty?
```
Comment (twinge) [inline app/models/insight_analyses.rb]: I'd refactor both conditions into one line:
```ruby
if recommends && unless recommends['rowset']['Row']
  recommends['rowset']['Row'].each { |c, _val|
          contacts_list.push(c['Column8'])
  }
end
```

File: config/config.travis.yml [hunk 2]
```diff
@@ -12,6 +12,6 @@ test:
   s3_secret: asdf
   s3_bucket: asdf
   itg_auth_key: asdf
-  obiee_key: 'MPDXWEBAPP'
```
Comment (twinge) [inline config/config.travis.yml]: This key/secret is now public in the github history. You'll want to change it if you haven't already.

File: config/config.travis.yml [hunk 3]
```diff
@@ -12,6 +12,6 @@ test:
...
-  obiee_secret: 'mHXbAAzzB08v'
-  obiee_base_url: 'https://insight.cru.org/analytics-ws/saw.dll?SoapImpl='
```
Comment (twinge) [inline config/config.travis.yml]: No harm in having a url be publicly visible

File: app/models/insight_analyses.rb (hunk 1 above)
Comment (LaelWatkins) [inline app/models/insight_analyses.rb]: makes sense and done.

File: config/config.travis.yml (hunk 2 above)
Comment (LaelWatkins) [inline config/config.travis.yml]: it's changed

File: config/config.travis.yml (hunk 3 above)
Comment (LaelWatkins) [inline config/config.travis.yml]: this is changed too.

Comment (LaelWatkins) [inline config/config.travis.yml]: Any additional suggestions on this PR? :-)

On Tue, Feb 17, 2015 at 5:08 PM, Josh Starcher <notifications@github.com>
wrote:

> In config/config.travis.yml
> <https://github.com/CruGlobal/mpdx/pull/280#discussion_r24859336>:
>
> > @@ -12,6 +12,6 @@ test:
> >    s3_secret: asdf
> >    s3_bucket: asdf
> >    itg_auth_key: asdf
> > -  obiee_key: 'MPDXWEBAPP'
> > -  obiee_secret: 'mHXbAAzzB08v'
> > -  obiee_base_url: 'https://insight.cru.org/analytics-ws/saw.dll?SoapImpl='
>
> No harm in having a url be publicly visible
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CruGlobal/mpdx/pull/280/files#r24859336>.
>



-- 
*Lael Watkins*
Senior Programmer Analyst
407 826 2548
lael.watkins@cru.org

**Comment by twinge** [comment]
:+1:

**Pull request merged**